README.md

```diff
@@ -208,72 +64,60 @@ Happo loads configuration in one of the following ways:
 ### Example configuration
 
 ```js
+var FirefoxTarget = require('happo-target-firefox');
+var S3Uploader = require('happo-uploader-s3');
+
 module.exports = {
-  // Control the interface on which the local server listens (defaults to 'localhost')
-  // (default: 'localhost')
-  bind: '0.0.0.0',
+  // the name of the summary file
+  // (default is shown below)
+  resultSummaryFilename: 'resultSummary.json',
+  
+  // A function that returns an "Uploader" interface for CI.
+  // (default: null)
+  uploader: () => new S3Uploader(),
 
-  // Control the port used for the local server
-  // (default: 4567)
-  port: 7777,
+  // Specify the folder where snapshots are saved
+  // (default: 'snapshots')
+  snapshotsFolder: 'happo-snapshots',
 
-  // List javascript source files. These can be files or raw URLs.
+  // An array of "targets" to run. Targets specify the environment to run
+  // the snapshots in. Must specify at least one.
   // (default: [])
-  sourceFiles: [
-    'https://unpkg.com/jquery@3.1.1',
-    'application.js',
-    'happo-examples.js',
+  targets: [
+    new FirefoxTarget({
+      // ... configuration for FirefoxTarget
+    }),
   ],
+};
+```
 
-  // List css source files. These can also be files or raw URLs.
-  // (default: [])
-  stylesheets: [
-    'application.css',
-  ],
 
-  // List directories where public files are accessible (useful for e.g. font files)
-  // (default: [])
-  publicDirectories: [
-    'public',
-  ],
+## Targets
 
-  // Specify the folder where snapshots are saved
-  // (default: 'snapshots')
-  snapshotsFolder: 'happo-snapshots',
+At the moment, `happo` only has one target, but more are coming!
 
-  // Configure the window size when taking snapshots
-  // (defaults shown below)
-  viewports: {
-    large: {
-      width: 1024,
-      height: 768,
-    },
-    medium: {
-      width: 640,
-      height: 888,
-    },
-    small: {
-      width: 320,
-      height: 444,
-    },
-  },
-};
-```
+- [`happo-target-firefox`](packages/happo-target-firefox)
+
+## Uploaders
+
+At the moment, `happo` only has one uploader, but more are coming!
+
+- [`happo-uploader-s3`](packages/happo-uploader-s3)
 
-## Command line tools
+## Command line API
 
-### `happo run`
+### `happo run [<targetName>]`
 
-This command will fire up a Firefox instance and take snapshots of all your
-Happo examples.
+This command will run all of the snapshots in the provided target name. If target
+name is omitted, then all targets are run.
 
 ### `happo review`
 
 Once `happo run` has finished, run `happo review` from the command line. This
 will open a page that compares the latest run's snapshots against the previous
 snapshots.
 
-### `happo debug`
+### `happo debug <targetName>`
 
 If you want to debug rendering your examples, you can run `happo debug`. This
 will open a browser window pointing at `/debug`, listing all your examples. If
```

Comment:
Some of these details should maybe be updated or removed. That can wait until later though.